### PR TITLE
Signup

### PR DESCRIPTION
**Pull request by Loges2108**

### CODE REVIEW

File: controllers/admin/customerController.js [hunk 1]
```diff
@@ -39,14 +39,9 @@ exports.adminLogin = async (req, res, next) => {
...
-      if (user.role === "customer") {
-        const error = new Error("Invalid credentials entered!");
-        error.statusCode = 500;
-        throw error;
-      }
-
       var userObj = {
         userId: user._id,
+        email:user.email,
         phoneNumber: user.phoneNumber,
```
Comment (kumuthac23) [inline controllers/admin/customerController.js:45]: phonenumber not used in this.so remove from this

File: routes/customer.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ const use = (fn) => (req, res, next) =>
 
 //Admin
 router.post("/adminLogin", use(adminCustomersController.adminLogin));
-router.post("/signup", use(adminCustomersController.signup));
+router.post("/signup", use(adminCustomersController.signUp));
```
Comment (kumuthac23) [inline routes/customer.js:11]: give correct spell for signup in both place

Comment (Loges2108) [reply routes/customer.js]: resolved

File: controllers/admin/customerController.js (hunk 1 above)
Comment (Loges2108) [reply controllers/admin/customerController.js]: removed it

**Pull request merged**